ddocs/medic/_attachments/translations/messages-en.properties

```diff
@@ -588,6 +588,7 @@ permission = Permission
 yes = Yes
 no = No
 confirm.destructive.navigation.forms = This form is not finished. You will lose your data if you leave now. A form is only complete when you hit Submit. Are you sure you want to leave?
+confirm.verification = Are you sure you want to verify this report as "{{proposedVerificationState}}"?
```

Comment:
As we only show this dialog when users cannot edit the verification I think we should add "This operation cannot be undone" or similar at the end.

Reply:
I agree. @amandacilek Any additional input on this copy?